newIDE/app/src/UI/SearchBar.js

```diff
@@ -299,6 +303,9 @@ const SearchBar = React.forwardRef<Props, SearchBarInterface>(
                     <Autocomplete
                       id={id}
                       options={tags}
+                      filterOptions={(options, state) =>
```

Comment:
I think using the limit option of createFilterOptions will avoid yet-another-function.